{"@t":"2023-12-16T06:40:09.2053157Z","@mt":"Starting web host"}
{"@t":"2023-12-16T06:40:14.6926439Z","@mt":"Now listening on: {address}","address":"https://localhost:7149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T06:40:14.7027264Z","@mt":"Now listening on: {address}","address":"http://localhost:5149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T06:40:14.7075541Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T06:40:14.7098889Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T06:40:14.7120312Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\Users\\atmanad\\Downloads\\Component-2\\Backend\\TweetApp.Backend\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T06:42:11.1864128Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"ba7f2f8e-b652-4aef-8cc6-d64f89b7684d","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU07E5L1II:00000013","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU07E5L1II"}
{"@t":"2023-12-16T06:42:27.7577769Z","@mt":"An error occurred using the connection to database '{database}' on server '{server}'.","@l":"Error","database":"db-tweet","server":"tcp:server-tweet.database.windows.net,1433","EventId":{"Id":20004,"Name":"Microsoft.EntityFrameworkCore.Database.Connection.ConnectionError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Connection","ActionId":"ba7f2f8e-b652-4aef-8cc6-d64f89b7684d","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU07E5L1II:00000013","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU07E5L1II"}
{"@t":"2023-12-16T06:42:28.0074341Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Database 'db-tweet' on server 'server-tweet.database.windows.net' is not currently available.  Please retry the connection later.  If the problem persists, contact customer support, and provide them the session tracing ID of '{C076B3B6-B81E-4ABC-8C4B-9D69F55D45BA}'.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:43f35f83-811e-4a2c-9cfd-26e0509c12fe\r\nError Number:40613,State:1,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"System.InvalidOperationException: An exception has been raised that is likely due to a transient failure. Consider enabling transient error resiliency by adding 'EnableRetryOnFailure' to the 'UseSqlServer' call.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Database 'db-tweet' on server 'server-tweet.database.windows.net' is not currently available.  Please retry the connection later.  If the problem persists, contact customer support, and provide them the session tracing ID of '{C076B3B6-B81E-4ABC-8C4B-9D69F55D45BA}'.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CheckPoolBlockingPeriod(Exception e)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenInternalAsync(Boolean errorsExpected, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\nClientConnectionId:43f35f83-811e-4a2c-9cfd-26e0509c12fe\r\nError Number:40613,State:1,Class:20\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"ba7f2f8e-b652-4aef-8cc6-d64f89b7684d","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU07E5L1II:00000013","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU07E5L1II"}
{"@t":"2023-12-16T06:42:28.2109658Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["18726.1856"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/register","StatusCode":200,"Elapsed":18726.1856,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"ba7f2f8e-b652-4aef-8cc6-d64f89b7684d","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU07E5L1II:00000013","ConnectionId":"0HMVU07E5L1II"}
{"@t":"2023-12-16T07:14:05.9631630Z","@mt":"Starting web host"}
{"@t":"2023-12-16T07:14:07.7416149Z","@mt":"Now listening on: {address}","address":"https://localhost:7149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T07:14:07.7651608Z","@mt":"Now listening on: {address}","address":"http://localhost:5149","EventId":{"Id":14,"Name":"ListeningOnAddress"},"SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T07:14:07.7701688Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T07:14:07.7722980Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T07:14:07.7743125Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\Users\\atmanad\\Downloads\\Component-2\\Backend\\TweetApp.Backend\\","SourceContext":"Microsoft.Hosting.Lifetime"}
{"@t":"2023-12-16T07:14:20.7378741Z","@mt":"Entity Framework Core {version} initialized '{contextType}' using provider '{provider}:{providerVersion}' with options: {options}","version":"6.0.8","contextType":"ApplicationDbContext","provider":"Microsoft.EntityFrameworkCore.SqlServer","providerVersion":"6.0.8","options":"None","EventId":{"Id":10403,"Name":"Microsoft.EntityFrameworkCore.Infrastructure.ContextInitialized"},"SourceContext":"Microsoft.EntityFrameworkCore.Infrastructure","ActionId":"07994f38-210b-4845-9107-43b81c8ae185","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU0QC2DDEQ:00000001","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU0QC2DDEQ"}
{"@t":"2023-12-16T07:14:22.4969685Z","@mt":"Failed executing DbCommand ({elapsed}ms) [Parameters=[{parameters}], CommandType='{commandType}', CommandTimeout='{commandTimeout}']{newLine}{commandText}","@l":"Error","elapsed":"998","parameters":"@__username_0='?' (Size = 100)","commandType":"Text","commandTimeout":30,"newLine":"\r\n","commandText":"SELECT TOP(1) [u].[Email], [u].[FirstName], [u].[LastName], [u].[Password]\r\nFROM [Users] AS [u]\r\nWHERE [u].[Email] = @__username_0","EventId":{"Id":20102,"Name":"Microsoft.EntityFrameworkCore.Database.Command.CommandError"},"SourceContext":"Microsoft.EntityFrameworkCore.Database.Command","ActionId":"07994f38-210b-4845-9107-43b81c8ae185","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU0QC2DDEQ:00000001","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU0QC2DDEQ"}
{"@t":"2023-12-16T07:14:22.5764449Z","@mt":"An exception occurred while iterating over the results of a query for context type '{contextType}'.{newline}{error}","@l":"Error","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid object name 'Users'.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()\r\nClientConnectionId:9c2d81a7-c1f8-4c76-b8e8-1ab3dfcc6f47\r\nError Number:208,State:1,Class:16\r\nClientConnectionId before routing:7b114bff-9940-4c7d-85a4-4392664b1aaa\r\nRouting Destination:be2a9930b512.tr2179.indiacentral1-a.worker.database.windows.net,11008","contextType":"TweetApp.Backend.Repository.ApplicationDbContext","newline":"\r\n","error":"Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid object name 'Users'.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__188_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(AsyncEnumerator enumerator, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()\r\nClientConnectionId:9c2d81a7-c1f8-4c76-b8e8-1ab3dfcc6f47\r\nError Number:208,State:1,Class:16\r\nClientConnectionId before routing:7b114bff-9940-4c7d-85a4-4392664b1aaa\r\nRouting Destination:be2a9930b512.tr2179.indiacentral1-a.worker.database.windows.net,11008","EventId":{"Id":10100,"Name":"Microsoft.EntityFrameworkCore.Query.QueryIterationFailed"},"SourceContext":"Microsoft.EntityFrameworkCore.Query","ActionId":"07994f38-210b-4845-9107-43b81c8ae185","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU0QC2DDEQ:00000001","RequestPath":"/api/v1/tweets/register","ConnectionId":"0HMVU0QC2DDEQ"}
{"@t":"2023-12-16T07:14:22.8000866Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["3785.8459"],"RequestMethod":"POST","RequestPath":"/api/v1/tweets/register","StatusCode":200,"Elapsed":3785.8459,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","ActionId":"07994f38-210b-4845-9107-43b81c8ae185","ActionName":"TweetApp.Backend.Controllers.UsersController.Register (TweetApp.Backend)","RequestId":"0HMVU0QC2DDEQ:00000001","ConnectionId":"0HMVU0QC2DDEQ"}
{"@t":"2023-12-16T07:29:27.1772099Z","@mt":"Starting web host"}
{"@t":"2023-12-16T07:29:27.4683442Z","@mt":"Host terminated unexpectedly","@l":"Fatal","@x":"Microsoft.Extensions.Hosting.HostFactoryResolver+HostingListener+StopTheHostException: Exception of type 'Microsoft.Extensions.Hosting.HostFactoryResolver+HostingListener+StopTheHostException' was thrown.\r\n   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)\r\n   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)\r\n   at Microsoft.Extensions.Hosting.HostBuilder.Build()\r\n   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()\r\n   at TweetApp.Backend.Program.Main(String[] args) in C:\\Users\\atmanad\\Downloads\\Component-2\\Backend\\TweetApp.Backend\\Program.cs:line 106"}
